src/Microsoft.ApplicationInsights/Channel/InMemoryTransmitter.cs

```diff
@@ -138,9 +140,27 @@ private void DequeueAndSend(TimeSpan timeout)
                     // send request
                     this.Send(telemetryItems, timeout).Wait();
                 }
-                catch (Exception e)
+                catch (AggregateException aex)
```

Comment:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/Test/ServerTelemetryChannel.Test/Shared.Tests/ServerTelemetryChannelE2ETests.cs#L162

There are few tests in SeverChannel E2e - they are not run in CI due to not stable when run parallel.

But you can run locally, and modiy it to validate the new exception unwinding based message. its okay to run/validate this when the cert issue is still occuring.